pyOCD/svd.py

```diff
@@ -0,0 +1,60 @@
+"""
+ mbed CMSIS-DAP debugger
+ Copyright (c) 2015 ARM Limited
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import threading
+from cmsis_svd.parser import SVDParser
```

Comment:
Note that the `cmsis_svd` package is currently quite large as it includes a collection of SVD files themselves.  If this a problem, we can probably figure something out.  Just create an issue with upstream: https://github.com/posborne/cmsis-svd (I'm the maintainer).